app/src/main/AndroidManifest.xml

```diff
@@ -62,14 +60,15 @@
 
         <activity
             android:name=".map.MapsActivity"
-            android:label="@string/title_activity_maps"/>
+            android:label="@string/title_activity_maps" />
 
         <service
             android:name=".map.BackgroundLocation"
             android:exported="true"
             android:permission="android.permission.BIND_JOB_SERVICE" />
 
-        <activity android:name=".questions.DisplayQuestionActivity"/>
+        <activity android:name=".questions.DisplayQuestionActivity" />
+        <activity android:name=".items.DisplayItemActivity"></activity>
```

Comment:
Why does the activity deosnt have the same syntax as the other ones? (/> becomes ></activity>) Is it the same?

Reply:
Yes, it's the same, just like html.